federation/pkg/federatedtypes/hooks.go

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
It doesn't appear this file is required now that the methods are only defined in the interface.